### PR TITLE
Updating content

### PR DESCRIPTION
**Pull request by Jaguaras**
Updating FAQ, Admin UI and removing outdated information.

## Summary by CodeRabbit

- **Documentation**
  - Updated `staking-FAQ.md` to reflect changes in the HOPR protocol, including the discontinuation of NFTs (except Network Registry NFT), introduction of Cover Traffic for rewards distribution, and impact of staking thresholds on rewards.
  - Simplified instructions in `using-hopr-admin-v2.md` for starting HOPR Admin by removing the need for a tmux session.
  - Added and clarified information in `waitlist-FAQ.md` about the Network Registry NFT, minimum stake requirements, updated waitlist schedule, and factors affecting waitlist positions.

### CODE REVIEW

File: versioned_docs/version-v2.1.0/node/staking-FAQ.md [hunk 1]
```diff
@@ -5,12 +5,102 @@ title: Staking FAQ
 
 ### How will the boost NFTs from previous releases be used in the future?
 
-The initial idea of the HOPR protocol was about staking and receiving rewards for relaying data. As a result, we are planning to discontinue all NFTs except the Network Registry NFT, which is necessary to meet the minimum staking threshold of 10,000 HOPR.
+The initial idea of the HOPR protocol was to stake and receive rewards for relaying data. All NFTs are discontinued except the Network Registry NFT, which is necessary to meet the minimum staking threshold of 10,000 HOPR. The Network Registry NFT doesn't provide any APY boost.
```
Comment (mjadach-iv) [inline versioned_docs/version-v2.1.0/node/staking-FAQ.md:8]:
```suggestion
All NFTs are discontinued except the Network Registry NFT, which is necessary to meet the minimum staking threshold of 10,000 HOPR. The Network Registry NFT doesn't provide any APY boost.
```

The 'The initial idea of the HOPR protocol was to stake and receive rewards for relaying data. ' suggests that the initial idea changed, but it did not. The sentense is unrelated to the question imo

File: versioned_docs/version-v2.1.0/node/staking-FAQ.md [hunk 2]
```diff
@@ -5,12 +5,102 @@ title: Staking FAQ
...
+- EconomicHandler applies the economic model and pushes the data to the queue.
+- Postman takes the queue and distributes a series of 1-hop messages to every node.
+
+### How often Cover Traffic is distributing the rewards?
+
+Currently, Cover Traffic (CT) distribution happens every hour. If your node is not reachable by our CT netwatchers during the rewards distribution, you will lose the rewards for the current distribution cycle and will need to wait for the next one. In the future, CT rewards distribution will be continuous.
+
+### Does Cover Traffic have threshold rewards caps, and if so, why?
+
+Cover Traffic has a minimum threshold cap of 10,000 wxHOPR. This means that if your total stake is equal to or greater than 10,000 wxHOPR, you will start receiving rewards, with the condition that your node remains reachable and performs normally.
```
Comment (mjadach-iv) [inline versioned_docs/version-v2.1.0/node/staking-FAQ.md:29]: What about when there is no NFT?

File: versioned_docs/version-v2.1.0/node/staking-FAQ.md [hunk 3]
```diff
@@ -5,12 +5,102 @@ title: Staking FAQ
...
+
+**Expected tickets** - Expected ticket numbers to be rewarded by the Cover traffic. 1 ticket equals to 1 wxHOPR.
+
+**APY (pct)** - Your staking rewards APY (Annual Percentage Yield).
+
+:::
+
+**1st scenario:**
+
+Every node runner are in between the Cover traffic thresholds, receiving maximum rewards based on their stake amount and node performance.
```
Comment (mjadach-iv) [inline versioned_docs/version-v2.1.0/node/staking-FAQ.md:65]:
```suggestion
Every node runner is in between the Cover traffic thresholds, receiving maximum rewards based on their stake amount and node performance.
```

File: versioned_docs/version-v2.1.0/node/waitlist-FAQ.md [hunk 4]
```diff
@@ -46,4 +46,4 @@ New slots are distributed on a tri-weekly basis, every Wednesday.
 
 ### Why did my position change after the waitlist was updated?
 
-Your position may change depending on whether the new applicant has an NFT and if the staking amount is higher.
\ No newline at end of file
+Your position may change if you don't have Network Registry NFT and new applicants joined with a higher stake and others may have withdrawn funds or been off-boarded due to a lack of funds.
```
[the remark this answers is not in the document]
Comment (mjadach-iv) [reply versioned_docs/version-v2.1.0/node/waitlist-FAQ.md:49]: good one

**Review by mjadach-iv — CHANGES_REQUESTED** [review]

File: versioned_docs/version-v2.1.0/node/staking-FAQ.md (hunk 2 above)
Comment (Jaguaras) [reply versioned_docs/version-v2.1.0/node/staking-FAQ.md]: Good point, I have updated on my latest commit.

**Review by mjadach-iv — APPROVED** [review]

**Pull request merged**